back/engines/commercial/analysis/app/lib/analysis/auto_tagging_method/nlp_topic.rb

```diff
@@ -50,5 +55,18 @@ def parse_topic_modeling_response(response)
     def max_topics(inputs_count)
       [inputs_count, (Math.log(inputs_count, 5) * 6).ceil].min
     end
+
+    def fit_inputs_in_context_window(inputs, project_title)
```

Comment:
I think it would be worth explaining that the window size is computed as the sum of the prompt length and the response length (in tokens).